spring-cloud-gcp-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -10,9 +10,58 @@ There are three entries in http://start.spring.io/[Spring Initializr] for Spring
 - GCP Messaging
 - GCP Storage
 
-The GCP Support entry contains autoconfiguration support for every Spring Cloud GCP integration.
+The GCP Support entry contains auto-configuration support for every Spring Cloud GCP integration.
 Most of the autoconfiguration code is only enabled if other dependencies are added to the classpath.
 
+|===
+|Spring Cloud GCP Starter |Required dependencies
+
+|Config
+|_None_
+
+|Spanner
+|- org.springframework.cloud:spring-cloud-gcp-data-spanner
+
+- org.springframework:spring-tx
+
+|Logging
+|- ch.qos.logback.contrib:logback-json-classic
```

Comment:
I agree. Documenting those four dependencies was a bit odd. 

To be clear what I have in mind with GCP support was a way for you to let users try features that aren't first class on start.spring.io. The majority of the work around the single autoconfigure module is related to this.

So, typically, I'd have expected you to write something like "If you have an app that uses JDBC with mysql or postgresql, then you can add GCP support and the xyz factory and we'll provide this and that".

What's the purpose of the GCP entry if such documentation doesn't exist?

Reply:
The logging starter already brings in `GCP Support` and is what should be in Spring Initializr.
It would be weird to ask people to import starters, if that's what should be selectable from Initializr.

@snicoll maybe it's because the GCP Support entry was your idea and not ours :sweat_smile:. From my/our side, it always made more sense to add the single individual starters instead of the general GCP Support entry.
Using that general entry, it seems like this is the only way.

Reply:
In any case, I don't think this issue relates to this PR, but rather to a possible issue with the logging module. All good to merge this?